src/codeenginestudio/thuy/Exercise1.java

```diff
@@ -1,23 +1,30 @@
 package codeenginestudio.thuy;
 
+import java.util.Arrays;
+
 public class Exercise1 {
 
     public static void main(String args[]) {
 
-        String s1 = "is2 Thi1s T4est 3a";
+        String s1 = "is2 Thi1s T4est 3a thu hoa";
 
         String[] sArray = splitString(s1);
 
-        int[] newArray = new int[sArray.length];
+        String[] newArray = new String[sArray.length];
 
         for (int i = 0; i < sArray.length; i++) {
 
             char[] intArray = sArray[i].toCharArray();
 
-            System.out.println(intArray);
+            int index = getNumberfromString(intArray);
+            System.out.println(index);
+
+            newArray[index] = sArray[i];
+
+            System.out.println(newArray[index]);
 
-            System.out.println(getNumberfromString(intArray));
         }
+        System.out.println("hhhhhhhhhhh"+ Arrays.toString(newArray));
```

Comment:
Please remove "hhhhhhh" and may be you can replace by "Output".

Reply:
Thank you your comment!